src/components/dropdown/dropdown.md

```diff
@@ -44,4 +44,40 @@ const theme = require('./dropdown-themes.scss');
 </div>
 ```
 
+Icons options version
+```js
+const theme = require('./dropdown-themes.scss');
+const areaIcon = require('./assets/icon_area_chart.svg');
+const arrowIcon = require('./assets/dropdown-arrow.svg');
+const infoIcon = require('./assets/info.svg');
+const data = [
+  { label: 'line', value: 'line' },
+  { label: 'area', value: 'area' },
+  { label: 'buble', value: 'buble '}
+]
+const icons = {
+'line': areaIcon,
+'area': arrowIcon,
+'buble': infoIcon
+}
+
+initialState = {
+  selected: data[0],
+  data: data
+}
+const onValueChange = (selected) => {
+  setState({ selected })
+}
+<div>
+  <Dropdown
+    theme={{ wrapper: theme.iconDropdownWrapper}}
+    value={icons[state.selected.value]}
+    options={state.data}
+    onValueChange={onValueChange}
+    iconsDropdown
```

Comment:
You could actually check if icons prop exists. You don't need an extra boolean

Reply:
done ⚡️